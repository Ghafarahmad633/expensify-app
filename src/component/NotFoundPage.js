import {Link} from "react-router-dom";
import React from "react";

const NotFoundPage=()=>(
    <div>
        <h1>Not Found!!</h1>
        <Link to="/">Go to Home</Link>
        <Link to="/help">Go to Help</Link>
    </div>
)
export default NotFoundPage