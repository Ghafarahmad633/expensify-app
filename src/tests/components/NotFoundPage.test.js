import React from 'react'
import NotFoundPage from '../../component/NotFoundPage'
import {shallow} from 'enzyme'
test('should render ExpenseList with expenses',()=>{
    const wraper =shallow(<NotFoundPage/>)
    expect(wraper).toMatchSnapshot()
})