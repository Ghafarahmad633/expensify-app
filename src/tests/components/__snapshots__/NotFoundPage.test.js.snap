// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseList with expenses 1`] = `
<div>
  <h1>
    Not Found!!
  </h1>
  <Link
    replace={false}
    to="/"
  >
    Go to Home
  </Link>
  <Link
    replace={false}
    to="/help"
  >
    Go to Help
  </Link>
</div>
`;
